zerver/lib/subscription_info.py

```diff
@@ -431,6 +431,7 @@ def get_stream_id(sub_dict: RawSubscriptionDict) -> int:
 
     traffic_stream_ids = {get_stream_id(sub_dict) for sub_dict in sub_dicts}
     recent_traffic = get_streams_traffic(stream_ids=traffic_stream_ids)
+    assert recent_traffic is not None
```

Comment:
I see that this is true when we don't pass a realm, and it's not a regression since that's how things worked previously.